layouts/api/arguments_table.ejs

```diff
@@ -1,4 +1,4 @@
-<table class="arguments">
+<table class="code-doc__arguments">
```

Comment:
We absolutely can, I just thought 'code-doc' was more globally usable if we plan for this to be consumable on other websites. Whereas 'api' is rather specific to this use case.

What would you think of just 'doc'? Same amount of letters but also more nonspecific while still getting the original idea across

Reply:
I'm struggling to imagine where we'd use these styles for something _other than_ API documentation, since they envision arguments and return values and whatnot. I don't think that `doc` gets the point across, since that could apply to the entire site.

Reply:
That's a valid point, especially `doc` applying the this entire site.

I'll go ahead and change it's prefix to `api`

Reply:
Alright and the prefix has been changed to `api` everywhere